### DISCUSSION TITLE
Hurray, with version 713, the ANR-rate dropped below the acceptable threshold!

### DISCUSSION THREAD

**Discussion by Klug76** [airsdk/Adobe-Runtime-Support#1563] [General]
Hurray, with version 713, the ANR-rate dropped below the acceptable threshold!
![anr_713_photo_2022-01-12_16-55-18](https://user-images.githubusercontent.com/1420061/149153955-68cbbe60-444c-4ce6-8608-0d3669367f0f.jpg)
Thanks to the AIR development team!

**Comment by marchbold** [comment]
Great to hear!!

**Comment by 2jfw** [comment]
Very nice - congratulations to you and Harman!
